scripts/tasks/how-to-examples.md

```diff
@@ -0,0 +1,38 @@
+---
+title: 'Examples'
+description: 'Electron examples directly for you to use'
+slug: examples
+hide_title: false
+---
+
+# Examples
+
+In this section we have documented some of the most popular things developers do
+in their Electron applications. The easiest way to run the examples is using
+[Fiddle][fiddle]. To do this, first you will have to download and install the
+right version for your Operating System from [here][fiddle].
+
+Once you have done this, you can press on the "Open in Fiddle" button that you
+will find below code samples like the following one:
+
+```fiddle docs/fiddles/quick-start
+window.addEventListener('DOMContentLoaded', () => {
+  const replaceText = (selector, text) => {
+    const element = document.getElementById(selector)
+    if (element) element.innerText = text
+  }
+
+  for (const type of ['chrome', 'node', 'electron']) {
+    replaceText(`${type}-version`, process.versions[type])
+  }
+})
+```
```

Comment:
I have an equivalent section meant for the first page you see when you go to `electronjs.org/docs`:

https://github.com/electron/electron/blob/bf04a5f9f83d2af602946e24a2465c724121d77d/docs/tutorial/introduction.md#running-examples-with-electron-fiddle

Reply:
@erickzhao what content do you think we should have as the landing of examples?